docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,124 @@ across groups and also the difference and ratio between the maximum and minimum:
+is an accurate representation of the world, but the world is unjust. 
+
+It's worth considering whether the assumptions underlying this measurement 
+model maintain construct validity (see :ref:`construct_validity`). 
+Consequential validity, for example, asks how the world shaped by using the 
+measurement model; in this case, the relevant question would be whether 
+demographic parity meets the criteria for establishing "fairness", itself 
+an unobservable theoretical construct, and whether satisfying demographic 
+parity brings us closer to the world we'd like to see. 
+
+Equalized odds
+~~~~~~~~~~~~~~
+A classifier :math:`h` satisfies equalized odds under a
+distribution over :math:`(X, A, Y)` if its prediction :math:`h(X)` is
+conditionally independent of the sensitive feature :math:`A` given the label
+:math:`Y`. This is equivalent to
+:math:`\E[h(X) \given A=a, Y=y] = \E[h(X) \given Y=y] \quad \forall a, y`.
+:footcite:`agarwal2018reductions` Equalized odds requires that the true 
+positive rate, :math:`\P(h(X)=1 | Y=1`, and the false positive rate, 
+:math:`\P(h(X)=1 | Y=0`, be equal across groups. 
+
+The inclusion of false positive rates acknowledges that different groups 
+experience different costs from misclassification. For example, in the case of 
+a model predicting a negative outcome (e.g., probability of recidivating) 
+that already disproportionately affects members of minority communities, 
+false positive predictions reflect preexisting disparities in outcomes 
+across minority and majority groups. Equalized odds further enforces that the 
+accuracy is equally high across all groups, punishing models that only 
+perform well on majority groups.
+
+Equalized odds and can be used to diagnose both allocation harms as well as 
+quality-of-service harms. It can be useful for diagnosing allocation harms 
+because its goal is to ensure that a machine learning model works equally 
+well for different groups. However, equalized odds makes the assumption 
+that the target variable :math:`Y` is a good measurement of the phenomena 
+being modeled, but that assumption may not hold if the measurement does not 
+satisfy the requirements of construct validity.
+
+Equal opportunity
+~~~~~~~~~~~~~~~~~
+Equal opportunity is a relaxed version of equalized odds that only considers
+conditional expectations with respect to positive labels, i.e., :math:`Y=1`.
+:footcite:`hardt2016equality` Another way of thinking about this metric is 
+requiring equal outcomes only within the subset of records belonging to the 
+positive class. For example, in the hiring example, equal opportunity 
+requires that the individuals who are actually hired have an equal opportunity 
+of being hired in the first place. However, by not considering whether false 
+positive rates are equivalent across groups, equal opportunity does not 
+capture the costs of missclassification disparities.
+
+Considerations for conditional group fairness
```

Comment:
Nitpick: this paragraph might make even more sense as a subparagraph of demographic parity. Though I suppose you could make similar arguments for equalized odds, it seems most natural to consider for demographic parity.

Reply:
Yeah, should we move it up to the demographic parity section?

Reply:
Sounds good!

Reply:
Ok, I will move the conditional group fairness section up to be a subsection of demographic parity.